codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/customizations/TimeSourceCustomization.kt

```diff
@@ -46,9 +47,9 @@ class TimeSourceCustomization(codegenContext: ClientCodegenContext) : ConfigCust
                         /// Sets the time source used for this service
                         pub fn time_source(
                             mut self,
-                            time_source: impl #{Into}<#{SharedTimeSource}>,
+                            time_source: impl #{IntoShared}<#{SharedTimeSource}>,
                         ) -> Self {
-                            self.set_time_source(#{Some}(time_source.into()));
+                            self.set_time_source(#{Some}(time_source.into_shared()));
```

Comment:
Why `into_shared` over `into`?

Reply:
The `Into` trait is imported in the prelude, so naming it `into` would conflict with that trait, which would lead to qualification being needed everywhere.